wp-content/themes/pub/wporg-learn-2020/archive-lesson-plan.php

```diff
@@ -40,9 +40,11 @@
 
 		<hr>
 		<?php
-		$categories = get_terms( 'wporg_lesson_category', array(
+		$categories = get_terms( array(
+			'taxonomy'   => 'wporg_lesson_category',
```

Comment:
Since 4.5.0, taxonomies should be passed via the ‘taxonomy’ argument in the $args array

https://developer.wordpress.org/reference/functions/get_terms/#description